src/_ert_job_runner/job.py

```diff
@@ -81,17 +81,38 @@ def run(self):
         environment = self.job_data.get("environment")
         if environment is not None:
             environment = {**os.environ, **environment}
-        proc = Popen(
-            arg_list,
-            stdin=stdin,
-            stdout=stdout,
-            stderr=stderr,
-            env=environment,
-        )
+
+        def ensure_file_handles_closed():
+            if stdin is not None:
+                stdin.close()
+            if stdout is not None:
+                stdout.close()
+            if stderr is not None:
+                stderr.close()
+
+        try:
+            proc = Popen(
+                arg_list,
+                stdin=stdin,
+                stdout=stdout,
+                stderr=stderr,
+                env=environment,
+            )
+            process = Process(proc.pid)
+        except OSError as e:
+            msg = f"{e.strerror} {e.filename}"
+            if e.strerror == "Exec format error" and e.errno == 8:
+                msg = (
+                    f"Missing execution format information "
```

Comment:
Very good detecting this very issue. A bit hard to read the message - maybe "clean it up"  into something like i.e 

```
f"Missing execution format information in file: '{e.filename}'."
f"Most likely you are missing and should add following to the top of the file: '#!/usr/bin/env python'"
```